### PR TITLE
sign_up

### PR DESCRIPTION
**Pull request by dungdnv-1036**
sign_up

### CODE REVIEW

**Comment by dungdnv-1036** [comment]
Screenshot from 2020-09-14 08-44-04

**Comment by dungdnv-1036** [comment]
Screenshot from 2020-09-14 08-54-48

**Comment by dungdnv-1036** [comment]
https://edu-redmine.sun-asterisk.vn/issues/29117

**Comment by dungdnv-1036** [comment]
https://edu-redmine.sun-asterisk.vn/issues/29117

**Comment by dungdnv-1036** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -26,8 +26,8 @@
         <div class="mega-dropdown-menu">
           <div class="w3ls_vegetables">
             <ul class="dropdown-menu drp-mnu">
-              <li><%= link_to t ".login" %></li>
-              <li><%= link_to t ".sign_up" %></li>
+              <li><%= link_to t(".login")%></li>
+              <li><%= link_to t(".sign_up"),signup_path %></li>
```
Comment (saobv-2223) [inline app/views/layouts/_header.html.erb]:
```suggestion
              <li><%= link_to t(".sign_up"), signup_path %></li>
```

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  <div class="w3_login_module">
+    <div class="module form-module">
+      <div class="toggle"><i class="fa fa-times fa-pencil"></i>
+      </div>
+      <div class="form">
+        <h2><%= t ".create_account"%></h2>
+        <%= form_for @user do |f| %>
+        <%= f.label t ".name" %>
+        <%= f.text_field :name, class: "form-control" %>
+        <%= f.label t ".email" %>
```
Comment (saobv-2223) [inline app/views/users/new.html.erb]: indent

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        <%= f.password_field :password_confirmation, class: "form-control" %>
+        <%= f.submit yield(:button_text), class: "btn btn-primary"%>
+        <% end %>
+      </div>
+    </div>
+  </div>
+</div>
+</div>
+<div class="clearfix"></div>
+</div>
```
Comment (saobv-2223) [inline app/views/users/new.html.erb]: ???? div

File: spec/requests/users_request_spec.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
```
Comment (saobv-2223) [inline spec/requests/users_request_spec.rb]: "

File: spec/views/users/create.html.erb_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (saobv-2223) [inline spec/views/users/create.html.erb_spec.rb]: "

File: spec/views/users/new.html.erb_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (saobv-2223) [inline spec/views/users/new.html.erb_spec.rb]: "

**Review by saobv-2223 — COMMENTED** [review]
CI fail nhé, CI pass mới được ready

**Comment by dungdnv-1036** [comment]
ready

**Pull request merged**